numpy/polynomial/polyutils.py

```diff
@@ -732,12 +732,12 @@ def _pow(mul_f, c, pow, maxpower):
 
     Parameters
     ----------
-    vander_f : function(array_like, int) -> ndarray
-        The 1d vander function, such as ``polyvander``
-    pow, maxpower :
-        See the ``<type>pow`` functions for more detail
     mul_f : function(array_like, array_like) -> ndarray
         The ``<type>mul`` function, such as ``polymul``
+    c : array_like
+        1-D array of array of series coefficients
+    pow, maxpower
+        See the ``<type>pow`` functions for more detail
```

Comment:
@charris any idea what happened here? Is this a copy-paste error from somewhere else?

Reply:
Looks like cut and paste error.

Reply:
Note that the empty `Examples` was deliberate and left for later filling in. Does it cause problems?

Reply:
> Note that the empty `Examples` was deliberate and left for later filling in. Does it cause problems?

I'm working on an [auto formatter](https://github.com/carreau/velin), and a side effect of how it work is that it's difficult to distinguish empty sections from section that do not exists when interacting with numpydoc's Docsscrape, I could add a "keep empty sections" if that's desired.

I'm also working on a terminal rendering of docstrings for IPython, for which not having empty section would also be better.

<img width="896" alt="Screen Shot 2021-01-23 at 15 38 37" src="https://user-images.githubusercontent.com/335567/106235638-ee4b1080-61af-11eb-8b53-f2766043b95e.png">

The other "issue" i'm encountering is the consistency around spacing in numpy docs, 
do you want (or not) a blank line between each parameter in the "Parameters" section, and also the indentation in see also which sometime is 

```
like : this aligned with the colon
       on the second line

and_some_time : like this where it's
   where its three or four spaces
```

Those are issues I might bring up at some points if you are interested in an auto formatter for numpydoc...